sdk/cosmos/azure-spring-data-cosmos/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-beans</artifactId>
-      <version>5.3.19</version> <!-- {x-version-update;org.springframework:spring-beans;external_dependency} -->
+      <version>5.3.20</version> <!-- {x-version-update;org.springframework:spring-beans;external_dependency} -->
```

Comment:
We have not updated spring-context and other dependencies, we should also update all of them to be in sync with version 5.3.20